osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/ConfirmDeviceRegistrationDataDto.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2015 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.valueobjects;
+
+import java.io.Serializable;
+
+public class ConfirmDeviceRegistrationDataDto implements Serializable {
```

Comment:
I don't know if this new class is useful. Sending the device type along with some other data doesn't seem to offer any benefit.

Reply:
It isn't particularly useful indeed. But on the other hand, would sending a confirm_register_device message without any content except it's header a good idea? So to keep in line with other messages I did add it. Indeed can be a design choice to add or not to add it.

Reply:
--> But on the other hand, would sending a confirm_register_device message without any content except it's header a good idea?  

If it is, I would say let's remove it ;-)

Reply:
I also added the dto to not to loose any info from the original confirm_register_device

Reply:
If there is information in the DTO that is needed (and not already available in the message metadata through the queues), it would be a reason to keep it.
We have other messages going through the layers without a Serializable message object, but only the message type and metadata.

It looks like the device type is included here. Is that something of interest that needs to be transferred as it was known where the DTO is created, to where it ends up?
If not, it may be better to drop this class.